lib/aida/skills/human_override.ex

```diff
@@ -38,7 +40,58 @@ defmodule Aida.Skill.HumanOverride do
       message |> Message.respond(clarification)
     end
 
-    def put_response(%{in_hours_response: response}, %{bot: bot} = message) do
+    defp weekday(datetime) do
+      datetime
+      |> DateTime.to_date
+      |> Date.day_of_week
+      |> Timex.day_shortname
+    end
+
+    def response(:in_hours, %{in_hours_response: in_hours_response}) do
+      in_hours_response
+    end
+
+    def response(:off_hours, %{off_hours_response: off_hours_response}) do
```

Comment:
Likewise, another one here like
```
def response(:off_hours, %{
  off_hours_response: nil,
  in_hours_response: in_hours_response
}), do: in_hours_response
```

would make the `off_hours_response` message optional too.